### ISSUE TITLE
Donate Crypto - Epic

### ISSUE THREAD

**Issue by willyogo** [Giveth/giveth-2#64]
As a user, I want to be able to donate with crypto.

[Mockups](https://www.figma.com/proto/FBqHg0UfeuaIueDEZG0tBS/Giveth-Design?node-id=1367%3A9&scaling=scale-down-width)

![Donate - Amount](https://user-images.githubusercontent.com/26232238/87249344-cff01400-c41b-11ea-938d-2360552d803e.png)

**Comment by markoprljic** [comment]
@willyogo One flaw I just noticed that we missed (or not) on Donation flow is to allow user to donate with Crypto on the first screen of the flow. User might be wondering "How can I donate crypto instead of USD?" and that only becomes obvious on the second step in the flow.

**Proposal**

1. The easiest way to fix this is to remove the first step as in original mockups https://www.figma.com/proto/FBqHg0UfeuaIueDEZG0tBS/Giveth-Design?node-id=1367%3A9&viewport=749%2C-2110%2C0.2276238054037094&scaling=min-zoom
2. Make the first step this version (I just mocked up now) https://www.figma.com/proto/FBqHg0UfeuaIueDEZG0tBS/Giveth-Design?node-id=1367%3A110&viewport=749%2C-2110%2C0.2276238054037094&scaling=min-zoom
3. When Crypto tab selected show these options https://www.figma.com/proto/FBqHg0UfeuaIueDEZG0tBS/Giveth-Design?node-id=1367%3A110&viewport=749%2C-2110%2C0.2276238054037094&scaling=min-zoom

Hope that makes sense and apologies for missing out this detail in the first place :)

**Comment by mateodaza** [comment]
Awesome! I'll follow these guidelines to update the current design

**Comment by willyogo** [comment]
We were thinking the same thing @markoprljic! When Mateo and I broke down the [Donate - Fiat](https://github.com/Giveth/giveth-2/issues/104) epic, we updated the flow so when a user clicks 'donate' they go straight to the 'Donate With' step. 'Credit Card' is selected by default and includes the 'Select Amount' functionality. If the user toggles to 'Donate With - Crypto' then they'll see the donation address/QR code.

![Donate to Project  - 4 (3)](https://user-images.githubusercontent.com/26232238/93898496-812ee800-fcb0-11ea-8d81-3124e7edf86c.png)

@mateodaza we will want to add the 'Select Amount' functionality for 'Donate With - Crypto' eventually, but we shouldn't worry about it in this initial MVP because users will donate crypto just by copying/scanning the address and sending from an external wallet. Once we add the wallet integrations that Marko mocked up previously (this is where onboard.js comes in) then we should add the amount input to the 'Donate With - Crypto' step like Marko just mocked up. 

Does that make sense?
